src/olympia/activity/models.py

```diff
@@ -595,6 +605,32 @@ def get_absolute_url(obj):
             '<a href="{0}">{1}</a>', get_absolute_url(self.user), self.user.name
         )
 
+        if _versions:
+            # Now that all arguments have been processed we can build a string
+            # for all the versions and build a string for addon that is
+            # specific to the reviewlog, with the correct channel for the
+            # review page link.
+            version = format_html(
+                ngettext('Version {0}', 'Versions {0}', len(_versions)),
+                # We're only joining already escaped/safe content.
+                mark_safe(', '.join(_versions)),
+            )
+
+        if channel is None and self.details and 'channel' in self.details:
+            channel = self.details['channel']
+
+        if type_ == 'reviewlog' and addon and addon_pk and addon_name:
```

Comment:
it's an edge case, but we can easily get the addon pk if we don't have an `addon` but have at least one version (and the name too if we want)

Reply:
Yeah... but at this point we're fighting the way `ActivityLog` builds that string... It doesn't keep instances around unless explicitly adding code for that, `addon` and `versions` are strings here, not instance(s) (that are built if one of the `arguments` is an `Addon` or `Version`, respectively).